src/pages/workspaces/workspace/WorkspaceContainer.ts

```diff
@@ -258,37 +258,39 @@ const WorkspaceAccessError = () => {
   ]);
 };
 
-interface WrapWorkspaceProps {
+export interface WrapWorkspaceOptions {
```

Comment:
Why are these options instead of props?

Reply:
For convention. `wrapWorkspace` is not a React component.

Reply:
What is its return type?

Reply:
`wrapWorkspace` returns a function that takes a component to be wrapped and returns a wrapper component.

Reply:
Is there a way to declare the return type?

Reply:
Perhaps. When I first looked at it though, to me it looked like a React component (which is a pretty loose definition). Seeing the return type specification may have helped me… though it's possible I wouldn't be able to grok it anyway. :(

Reply:
👍 Will add.

Reply:
Added a return type and a comment in https://github.com/DataBiosphere/terra-ui/pull/4507/commits/f064f48e682a75f407d06c90f25896c2abc6f64f.